CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Next release
 
+### Change to enum membership semantics
+
+As per the updated [typing specification for enums](https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members),
```

Comment:
I feel we can expand this a bit to explain the breakage, something like "Historically, enums defined in stub files were often written without the value for each member, such as "CAT: int". Mypy will now warn if it encounters an enum in a stub file without any members. If you encounter this warning, you may have to rewrite your enum to add values."